sycl/test/helpers.hpp

```diff
@@ -6,6 +6,7 @@
 //
 //===----------------------------------------------------------------------===//
 
+#include <iostream>
 #include <sycl/sycl.hpp>
```

Comment:
```suggestion
#include <sycl/sycl.hpp>

#include <iostream>
```
Let's include `iostream` after sycl.hpp to catch cases when sycl.hpp uses `iostream` functionality w/o including the header.